assignment1/src/main/java/ro/utcn/sd/he/assignment1/controller/CommandLineController.java

```diff
@@ -35,20 +33,21 @@ public void run(String... args){
         while(!done){
             print("Enter command:");
             String command = scanner.next().trim();
+            scanner.nextLine();     //dump the line
             done = handleCommand(command);
 
 
         }
     }
 
+
     private boolean handleCommand(String command){
         switch(command){
             case "search":
                 handleSearch();
                 return false;
             case "list":
                 List<Question> questions = questionService.listQuestions();
```

Comment:
You can do this with a lambda (you already did this before with a lambda).

Reply:
I can't really understand what you are referring to here :)